kore/app/share/GlobalMain.hs

```diff
@@ -270,6 +269,8 @@ the project
 data GlobalOptions = GlobalOptions
     { -- | Version flag [default=false]
       willVersion :: !Bool
+    , -- | Experimental simplifier flag [default=false]
+      simplifierx :: !Bool
```

Comment:
I'd rather create a new datatype instead of using `Bool`, `data ExperimentalSimplifier = Enabled | Disabled`.